### PR TITLE
Updated Changes

### PR DESCRIPTION
**Pull request by Shirish-Panchal-24**
## Summary by CodeRabbit

- **New Features**
	- Introduced a live server configuration setting for local development.
	- Added new HTML and CSS files for a structured visiting card layout.
	- Implemented a visually appealing card design with multiple sections for personal and organizational information.

- **Bug Fixes**
	- Corrected HTML structure for improved layout and presentation.

- **Documentation**
	- Updated HTML files to include metadata and links for styling and icons.

### CODE REVIEW

File: SubmittedAssignments/Shirish_Panchal_80/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <div class="left-small"></div>
+        <div class="col"></div>
+      </div>
+                <!-- Card2 Background -->
+      <div class="card2-background">
+        <div class="Right-big"></div>
+        <div class="Right-mid"></div>
+        <div class="Right-small"></div>
+      </div>
+
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Shirish_Panchal_80/index.html:18]: <div class="card-background"> close this div

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]

**Review by Shirish-Panchal-24 — COMMENTED** [review]
.

**Review by Adarshovhal — CHANGES_REQUESTED** [review]
Don't use px. Use only mm and pt

**Review by Adarshovhal — COMMENTED** [review]
Review Done

**Pull request closed without merging**